examples/impls/nnt-data-on-premise.yml

```diff
@@ -0,0 +1,164 @@
+name: ntt-data-on-premise
```

Comment:
the filename should start `ntt` not `nnt` and it should overwrite the file already in the example/impls directory